nncf/experimental/post_training/api/engine.py

```diff
@@ -83,11 +82,11 @@ def set_inputs_transforms(self, inputs_transforms: Callable):
         """
         self._inputs_transforms = inputs_transforms
 
-    def compute_statistics(self, statistics_layout: Dict[str, TensorStatisticCollectorBase]) -> None:
+    def compute_statistics(self, statistic_points: StatisticPointsContainer) -> None:
         """
         Performs model inference on specified dataset subset for statistics collection and input-based layers layout
 
-        :param statistics_layout: dictionary of stats collection functions {node_name: {stat_name: fn}}
+        :param statistic_points: StatisticsPoints in which statistics should be collected and registered.
         :return statistics: per-layer statistics for the further model optimization
```

Comment:
compute_statistics function has to return per-layer statistics for the further model optimization but the returned type hint is None. Could you please explain, the mistake in the type hint or doc string?

Reply:
Docstring was incorrect.